roles/confluent.common/tasks/redhat.yml

```diff
@@ -36,3 +36,8 @@
     name: "{{redhat_java_package_name}}"
     state: present
   when: install_java|bool
+
+- name: Install unzip
+  yum:
+    name: unzip
+    state: present
```

Comment:
Why are we install unzip here?

Reply:
That was part of nickoleta's confluent hub connector jar PR, the unarchive module needs unzip. Technically unzip should be moved into the connect role (i think). But im going to wait until unit tests are written to edit code